api/src/main/java/ca/bc/gov/educ/api/gradstudent/model/dto/GraduationStudentRecordDistribution.java

```diff
@@ -19,6 +19,8 @@ public class GraduationStudentRecordDistribution extends BaseModel{
     private String legalLastName;
     private String schoolOfRecord;
     private String schoolAtGrad;
```

Comment:
Can we get rid of schoolOfRecord and schoolAtGrad?

Reply:
Done